Explorer/Assets/DCL_RenderPass_GenerateSkyBox.cs

```diff
@@ -0,0 +1,136 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public partial class DCL_RenderFeature_ProceduralSkyBox : ScriptableRendererFeature
+{
+    public class DCL_RenderPass_GenerateSkyBox : ScriptableRenderPass
+    {
+        const string profilerTag = "Custom Pass: GenerateSkyBox";
+        private Material m_Material_Generate;
+        private ProceduralSkyBoxSettings_Generate m_Settings_Generate;
+        private Matrix4x4 viewMatrix;
+        private Matrix4x4 projMatrix;
+        RTHandle m_SkyBoxCubeMap_RTHandle;
+
+        private static readonly int s_ParamsID = Shader.PropertyToID("_CurrentCubeFace");
+        private static readonly int s_SunPosID = Shader.PropertyToID("_SunPos");
+        private static readonly int s_SunColID = Shader.PropertyToID("_SunColour");
+
+        // Constants
+        private const string k_SkyBoxCubemapTextureName = "_SkyBox_Cubemap_Texture";
+
+        // Statics
+        private static readonly int s_SkyBoxCubemapTextureID = Shader.PropertyToID(k_SkyBoxCubemapTextureName);
+        
+        private enum ShaderPasses
+        {
+            CubeMapFace_Right = 0,
+            CubeMapFace_Left = 1,
+            CubeMapFace_Up = 2,
+            CubeMapFace_Down = 3,
+            CubeMapFace_Front = 4,
+            CubeMapFace_Back = 5
+        }
+
+        internal DCL_RenderPass_GenerateSkyBox()
+        {
+            Debug.Log("DCL_RenderPass_GenerateSkyBox::Constructor");
+            Vector3 from = new Vector3(0.0f, 0.0f, 0.0f);
+            Vector3 to = new Vector3(1.0f, 0.0f, 0.0f);
+            Vector3 up = new Vector3(0.0f, 0.0f, 1.0f);
+            viewMatrix = Matrix4x4.LookAt(from, to, up);
+
+            float fov = 90.0f;
```

Comment:
Did you have a plan to expose them somehow or they are purely constant?

Reply:
No longer needed, will remove.